docs/src/pages/authentication.rst

```diff
@@ -60,4 +58,4 @@ would be DEMO (note the upper-case). i.e.:
 
 Server-side authentication
 --------------------------
-We strongly recommend using OAuth for authentication. In a development environment, you can configure the application to use the "Runner" credentials stored in environment variables. (``SA_USER``, ``SA_ROLE``, ``SA_DATABASE``, ``SA_WAREHOUSE``, ``PRIVATE_KEY``, ``PRIVATE_KEY_PASSWORD``).
+We strongly recommend using OAuth for authentication, but locally or in an otherwise secured environment, you can configure the application to use the same credentials stored in environment variables as the runners do (``SNOWFLAKE_ACCOUNT``, ``SA_USER``, ``SA_ROLE``, ``SA_DATABASE``, ``SA_WAREHOUSE``, ``PRIVATE_KEY``, ``PRIVATE_KEY_PASSWORD``).
```

Comment:
Let's make this warning more explicit. "Snowflake OAuth is the recommended authentication method. Storing credentials in the following environment variables will enable anyone with web access to manage the rules and data connectors. Please don't use these environment variables in production."